*** Snoopy.class.php.0	Mon Sep 15 18:21:27 2008
--- Snoopy.class.php	Sun Aug 14 15:14:44 2011
***************
*** 4,11 ****
  
  Snoopy - the PHP net client
  Author: Monte Ohrt <monte@ispi.net>
! Copyright (c): 1999-2000 ispi, all rights reserved
! Version: 1.01
  
   * This library is free software; you can redistribute it and/or
   * modify it under the terms of the GNU Lesser General Public
--- 4,11 ----
  
  Snoopy - the PHP net client
  Author: Monte Ohrt <monte@ispi.net>
! Copyright (c): 1999-2008 New Digital Group, all rights reserved
! Version: 1.2.4
  
   * This library is free software; you can redistribute it and/or
   * modify it under the terms of the GNU Lesser General Public
***************
*** 22,34 ****
   * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  
  You may contact the author of Snoopy by e-mail at:
! monte@ispi.net
! 
! Or, write to:
! Monte Ohrt
! CTO, ispi
! 237 S. 70th suite 220
! Lincoln, NE 68510
  
  The latest version of Snoopy can be obtained from:
  http://snoopy.sourceforge.net/
--- 22,28 ----
   * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  
  You may contact the author of Snoopy by e-mail at:
! monte@ohrt.com
  
  The latest version of Snoopy can be obtained from:
  http://snoopy.sourceforge.net/
***************
*** 48,54 ****
  	var $proxy_user		=	"";					// proxy user to use
  	var $proxy_pass		=	"";					// proxy password to use
  	
! 	var $agent			=	"Snoopy v1.2.3";	// agent we masquerade as
  	var	$referer		=	"";					// referer info to pass
  	var $cookies		=	array();			// array of cookies to pass
  												// $cookies["username"]="joe";
--- 42,48 ----
  	var $proxy_user		=	"";					// proxy user to use
  	var $proxy_pass		=	"";					// proxy password to use
  	
! 	var $agent			=	"Snoopy v1.2.4";	// agent we masquerade as
  	var	$referer		=	"";					// referer info to pass
  	var $cookies		=	array();			// array of cookies to pass
  												// $cookies["username"]="joe";
***************
*** 795,802 ****
  			$headers .= "User-Agent: ".$this->agent."\r\n";
  		if(!empty($this->host) && !isset($this->rawheaders['Host'])) {
  			$headers .= "Host: ".$this->host;
! //			if(!empty($this->port))
! //				$headers .= ":".$this->port;
  			$headers .= "\r\n";
  		}
  		if(!empty($this->accept))
--- 789,796 ----
  			$headers .= "User-Agent: ".$this->agent."\r\n";
  		if(!empty($this->host) && !isset($this->rawheaders['Host'])) {
  			$headers .= "Host: ".$this->host;
! 			if(!empty($this->port))
! 				$headers .= ":".$this->port;
  			$headers .= "\r\n";
  		}
  		if(!empty($this->accept))
***************
*** 1012,1019 ****
  		
  		$headerfile = tempnam($temp_dir, "sno");
  
! 		$safer_URI = strtr( $URI, "\"", " " ); // strip quotes from the URI to avoid shell access
! 		exec($this->curl_path." -D \"$headerfile\"".$cmdline_params." \"".$safer_URI."\"",$results,$return);
  		
  		if($return)
  		{
--- 1006,1012 ----
  		
  		$headerfile = tempnam($temp_dir, "sno");
  
! 		exec($this->curl_path." -k -D \"$headerfile\"".$cmdline_params." \"".escapeshellcmd($URI)."\"",$results,$return);
  		
  		if($return)
  		{
