*** qr_img.php.0	Mon Sep 15 18:21:27 2008
--- qr_img.php	Sun Aug 14 15:12:40 2011
***************
*** 1,23 ****
  <?php
- /* ========================================================================
-  *
-  * @license This source file is subject to version 3.01 of the PHP license,
-  *              that is available at http://www.php.net/license/3_01.txt
-  *              If you did not receive a copy of the PHP license and are unable 
-  *              to obtain it through the world-wide-web, please send a note to 
-  *              license@php.net so we can mail you a copy immediately.  
-  *
-  * @category   Application of MyNETS
-  * @project    OpenPNE UsagiProject 2006-2007
-  * @package    MyNETS
-  * @author     UsagiProject <info@usagi.mynets.jp>
-  * @copyright  2006-2007 UsagiProject <author member ad http://usagi.mynets.jp/member.html>
-  * @version    MyNETS,v 1.1.0
-  * @since      File available since Release 1.1.0 Nighty
-  * @chengelog  [2007/06/09] Ver1.1.0Nighty package
-  * ======================================================================== 
-  */
- 
  /*************
   #
  # QRcode image PHP scripts  version 0.50g (C)2000-2005,Y.Swetake
--- 1,4 ----
***************
*** 127,134 ****
  
    /*  --- determine encode mode */
  
!   if (ereg("[^0-9]",$qrcode_data_string)){
!     if (ereg("[^0-9A-Z \$\*\%\+\-\.\/\:]",$qrcode_data_string)) {
  
       /*  --- 8bit byte mode */
  
--- 108,117 ----
  
    /*  --- determine encode mode */
  
! /*if (ereg("[^0-9]",$qrcode_data_string)){
!     if (ereg("[^0-9A-Z \$\*\%\+\-\.\/\:]",$qrcode_data_string)) {*/
! if (preg_match("/[^0-9]/",$qrcode_data_string)!=0){
!     if (preg_match("/[^0-9A-Z \$\*\%\+\.\/\:\-]/",$qrcode_data_string)!=0) {
  
       /*  --- 8bit byte mode */
  
