*** AmazonECS4.php.0	Mon Sep 15 18:21:27 2008
--- AmazonECS4.php	Sun Aug 14 15:13:45 2011
***************
*** 35,49 ****
  * @category  Web Services
  * @package   Services_Amazon
  * @author    John Downey <jdowney@gmail.com>
! * @author    Tatsuya Tsuruoka <ttsuruoka@p4life.jp>
  * @copyright 2004 John Downey
  * @license   http://www.freebsd.org/copyright/freebsd-license.html 2 Clause BSD License
! * @version   CVS: $Id: AmazonECS4.php,v 1.4 2006/08/04 10:31:00 ttsuruoka Exp $
  * @link      http://pear.php.net/package/Services_Amazon/
  * @filesource
  */
  
  /**
  * Uses PEAR class for error management
  */
  require_once 'PEAR.php';
--- 35,56 ----
  * @category  Web Services
  * @package   Services_Amazon
  * @author    John Downey <jdowney@gmail.com>
! * @author    Tatsuya Tsuruoka <tatsuya.tsuruoka@gmail.com>
  * @copyright 2004 John Downey
  * @license   http://www.freebsd.org/copyright/freebsd-license.html 2 Clause BSD License
! * @version   CVS: $Id: AmazonECS4.php,v 1.9 2008/04/02 04:23:38 ttsuruoka Exp $
  * @link      http://pear.php.net/package/Services_Amazon/
  * @filesource
  */
  
  /**
+  * NOTICE:
+  * This class is for backward compatibility and should be considered obsolete.
+  * After August 15, 2009, all requests without a signature will be denied.
+  * You may as well use Services_Amazon when you create a new application.
+  */
+ 
+ /**
  * Uses PEAR class for error management
  */
  require_once 'PEAR.php';
***************
*** 91,99 ****
  *
  * @package Services_Amazon
  * @author  John Downey <jdowney@gmail.com>
! * @author  Tatsuya Tsuruoka <ttsuruoka@p4life.jp>
  * @access  public
! * @version Release: 0.7.1
  * @uses    PEAR
  * @uses    HTTP_Request
  * @uses    XML_Unserializer
--- 98,106 ----
  *
  * @package Services_Amazon
  * @author  John Downey <jdowney@gmail.com>
! * @author  Tatsuya Tsuruoka <tatsuya.tsuruoka@gmail.com>
  * @access  public
! * @version Release: 0.8.0
  * @uses    PEAR
  * @uses    HTTP_Request
  * @uses    XML_Unserializer
***************
*** 240,246 ****
      */
      function getApiVersion()
      {
!         return '0.7.1';
      }
  
      /**
--- 247,253 ----
      */
      function getApiVersion()
      {
!         return '0.8.0';
      }
  
      /**
***************
*** 888,894 ****
      function TransactionLookup($transaction_id, $options = array())
      {
          $params = $options;
!         $params['Operation'] = 'SimilarityLookup';
          $params['TransactionId'] = $transaction_id;
          return $this->_sendRequest($params);
      }
--- 895,901 ----
      function TransactionLookup($transaction_id, $options = array())
      {
          $params = $options;
!         $params['Operation'] = 'TransactionLookup';
          $params['TransactionId'] = $transaction_id;
          return $this->_sendRequest($params);
      }
