*** init.inc.0	Mon Sep 15 18:21:27 2008
--- init.inc	Sun Aug 14 16:01:12 2011
***************
*** 79,85 ****
      'dk.pdx.ne.jp',
      'wm.pdx.ne.jp',
      'emnet.ne.jp',
!     'i.softbank.ne.jp',     //20080714 iPhone用携帯アドレス追加
  );
  
  // Smarty の設定
--- 79,86 ----
      'dk.pdx.ne.jp',
      'wm.pdx.ne.jp',
      'emnet.ne.jp',
!     'i.softbank.jp',     //20080714 iPhone用携帯アドレス追加
!     					 //20090721 iPhone用アドレス記述ミスのため修正
  );
  
  // Smarty の設定
